tests/cross/src/main/scala/tests/BasePCSuite.scala

```diff
@@ -70,13 +70,16 @@ abstract class BasePCSuite extends BaseSuite {
     JdkSources().foreach(jdk => index.addSourceJar(jdk))
   }
 
-  override def munitNewTest(test: Test): Test = {
-    val testName =
-      if (isCI && BuildInfo.scalaCompilerVersion != BuildInfoVersions.scala212)
-        s"${BuildInfo.scalaCompilerVersion}-${test.name}"
-      else test.name
-    test.withName(testName)
-  }
+  override def munitTestTransforms
+      : List[TestTransform] = super.munitTestTransforms ++ List(
```

Comment:
minor, but can you manually break the line after `=` so that it does not break on the type annotation?

Reply:
also, no need to wrap this in a `List`, you can do

```scala
super.munitTestTransforms :+ new TestTransform(...)
```